pfcpiface/fastpath.go

```diff
@@ -9,12 +9,20 @@ import (
 )
 
 type fastPath interface {
+	/* Close any pending sessions */
 	exit()
+	/* setup internal parameters and channel with fastPath */
 	setUpfInfo(u *upf, conf *Conf)
+	/* set udp and pfcp conn parameters in interface */
 	setInfo(udpConn *net.UDPConn, udpAddr net.Addr, pconn *PFCPConn)
+	/* simulator mode setup */
 	sim(u *upf, method string)
-	sendMsgToUPF(method string, pdrs []pdr, fars []far) uint8
-	sendDeleteAllSessionsMsgtoUPF()
+	/* write pdr/far/qer to fastpath */
+	sendMsgToUPF(u *upf, method string, pdrs []pdr, fars []far) uint8
+	/* delete all pdrs/fars/qers/ installed in fastpath tabled */
+	sendDeleteAllSessionsMsgtoUPF(u *upf)
+	/* check of communication channel to fastpath is setup */
+	handleChannelStatus(u *upf) bool
```

Comment:
Can we use a better name? `isConnected()`? also why does it need access to upf? whatever the fastpath needs to check should already be within its internal structures.

Reply:
done